plugin/pom.xml

```diff
@@ -15,7 +15,7 @@
   <description>Adds support for building Dockerfiles in Maven</description>
 
   <properties>
-    <docker-client.version>8.8.1</docker-client.version>
+    <docker-client.version>8.11.0</docker-client.version>
```

Comment:
I'm releasing 8.11.1 right now. Can you update this once that's out?

Reply:
Sure! Done